db/templates/shell/dark.ejs

```diff
@@ -47,7 +47,7 @@ elif [ "${TERM%%-*}" = "screen" ]; then
   printf_template="\033P\033]4;%d;rgb:%s\007\033\\"
   printf_template_var="\033P\033]%d;rgb:%s\007\033\\"
   printf_template_custom="\033P\033]%s%s\007\033\\"
-elif [[ $- != *i* ]]; then
+elif ! [ -z "${-##*i*}" ]; then
```

Comment:
I'm not that proficient in bash scripting so I used what worked for me, I guess if `-n` negates the that's perfect